src/rdb/rdb_raft.c

```diff
@@ -78,6 +78,10 @@ rdb_raft_rc(int raft_rc)
 	}
 }
 
+/* kccain - raft calls:
+ *  raft_node_get_udata()
+ *  raft_node_get_id()
+ */
 static int
 rdb_raft_cb_send_requestvote(raft_server_t *raft, void *arg, raft_node_t *node,
 			     msg_requestvote_t *msg)
```

Comment:
I think every of these rdb_raft_cb_* callbacks is to be called holding d_raft_mutex, taken by rdb before calling into raft.